test/qctrl/test_qctrl.py

```diff
@@ -28,6 +39,90 @@
 DIFFERENCE_THRESHOLD = 0.35
 
 
+class TestV2PrimitivesQCTRL(IBMIntegrationTestCase):
+    """Integration tests for V2 primitives using QCTRL."""
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.bell = bell()
+        self.backend = self.service.least_busy(simulator=False)
+
+    @run_integration_test
+    def test_sampler_v2_qctrl(self, service):
+        """Test qctrl bell state with samplerV2"""
+        shots = 1
+        bell_circuit = QuantumCircuit(2)
+        bell_circuit.h(0)
+        bell_circuit.cx(0, 1)
+        bell_circuit.measure_all()
+
+        pm = generate_preset_pass_manager(backend=self.backend, optimization_level=1)
+        isa_circuit = pm.run(bell_circuit)
+
+        with Session(service, backend=self.backend):
+            sampler = SamplerV2()
+
+            result = sampler.run([isa_circuit], shots=shots).result()
+            self._verify_sampler_result(result, num_pubs=1)
+
+    @run_integration_test
+    def test_estimator_v2_qctrl(self, service):
+        """Test simple circuit with estimatorV2 using qctrl."""
+        backend = service.backend(self.backend)
```

Comment:
I don't think this would work because `self.backend` is already a backend object.

Reply:
Oops, missed that - all tests passing now